app/assets/sass/application.scss

```diff
@@ -757,4 +779,101 @@ ul.sub-tabs li a.active span.badge {
 .govuk-input__prefix {
     height: 40px;
     font-size: 19px;
-}
\ No newline at end of file
+}
+
+.view-claim-display-helper{
+    display: flex;
+    align-items: center;
+} 
+
+.justify-content-end {
```

Comment:
So not keen on this. Rules should reflect the block and what it does, not just be named after a specific CSS statement, better to create or extend the existing rule that this is being applied to. Like `view-claim-display-helper` is a good name.